File: specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-08-01/examples-js/liveoutput-list-all.js
const { AzureMediaServices } = require("@azure/arm-mediaservices");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Lists the live outputs of a live event.
 *
 * @summary Lists the live outputs of a live event.
 * x-ms-original-file: specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-08-01/examples/liveoutput-list-all.json
 */
async function listAllLiveOutputs() {
  const subscriptionId =
    process.env["MEDIASERVICES_SUBSCRIPTION_ID"] || "0a6ec948-5a62-437d-b9df-934dc7c1b722";
  const resourceGroupName = process.env["MEDIASERVICES_RESOURCE_GROUP"] || "mediaresources";
  const accountName = "slitestmedia10";
  const liveEventName = "myLiveEvent1";
  const credential = new DefaultAzureCredential();
  const client = new AzureMediaServices(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.liveOutputs.list(resourceGroupName, accountName, liveEventName)) {
    resArray.push(item);
  }
  console.log(resArray);
}

File: specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/examples-java/liveoutput-list-all.java
/** Samples for LiveOutputs List. */
public final class Main {
    /*
     * x-ms-original-file: specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/examples/liveoutput-list-all.json
     */
    /**
     * Sample code: List all LiveOutputs.
     *
     * @param manager Entry point to MediaServicesManager.
     */
    public static void listAllLiveOutputs(com.azure.resourcemanager.mediaservices.MediaServicesManager manager) {
        manager
            .liveOutputs()
            .list("mediaresources", "slitestmedia10", "myLiveEvent1", com.azure.core.util.Context.NONE);
    }
}
